### PR TITLE
Add Service NSW

### PR DESCRIPTION
**Pull request by jimmyvanau**

### CODE REVIEW

**Comment by Carlgo11** [comment]
@jimmyvanau your PRs don't show up on [CircleCI](https://app.circleci.com/pipelines/github/2factorauth/twofactorauth). Have you changed some settings to specifically opt out from CircleCI? :thinking:

**Comment by phallobst** [comment]
@jimmyvanau see https://circleci.com/docs/2.0/enable-checks/

**Comment by jimmyvanau** [comment]
Hmm, I'm not sure what I'm doing differently - it's not intentional.

**Comment by jimmyvanau** [comment]
I don't have any branch protection rules at https://github.com/jimmyvanau/twofactorauth/settings/branches

**Comment by phallobst** [comment]
Could you do an empty commit on this branch? Then we'll see if only the initial commit is affected or all of them.

**Comment by jimmyvanau** [comment]
Is it the branch?
```
jimmy@DESKTOP-LCM3GUJ MINGW64 /c/repos/twofactorauth (add-service-nsw-gov-au)
$ git commit --allow-empty -m "empty commit"
[add-service-nsw-gov-au db02c583] empty commit

jimmy@DESKTOP-LCM3GUJ MINGW64 /c/repos/twofactorauth (add-service-nsw-gov-au)
$ git push origin master
Everything up-to-date

jimmy@DESKTOP-LCM3GUJ MINGW64 /c/repos/twofactorauth (add-service-nsw-gov-au)
$ git log -n 5
commit db02c583824c085695c38a46c9d22fbde52c3a6d (HEAD -> add-service-nsw-gov-au)
Author: jimmyvanau <7769349+jimmyvanau@users.noreply.github.com>
Date:   Fri Nov 26 22:40:10 2021 +1100

    empty commit

commit 580bb33526207eeeb5b19864fc91f75916212a32 (origin/add-service-nsw-gov-au)
Author: jimmyvanau <7769349+jimmyvanau@users.noreply.github.com>
Date:   Wed Nov 24 19:41:38 2021 +1100

    Add Service NSW

commit e13d418073de36c97d3bb6c8b01760273b957298 (upstream/master, origin/master, origin/HEAD, master)
Author: sn00py1310 <61204194+sn00py1310@users.noreply.github.com>
Date:   Mon Nov 22 11:20:19 2021 +0100

    Add OpenWeather (#6295)

    * Add SVG and JSON

    * Update openweathermap.org.svg

:...skipping...
commit db02c583824c085695c38a46c9d22fbde52c3a6d (HEAD -> add-service-nsw-gov-au)
Author: jimmyvanau <7769349+jimmyvanau@users.noreply.github.com>
Date:   Fri Nov 26 22:40:10 2021 +1100

    empty commit

commit 580bb33526207eeeb5b19864fc91f75916212a32 (origin/add-service-nsw-gov-au)
Author: jimmyvanau <7769349+jimmyvanau@users.noreply.github.com>
Date:   Wed Nov 24 19:41:38 2021 +1100

    Add Service NSW
```

**Comment by jimmyvanau** [comment]
Wait silly me, pushed to wrong branch, I pushed commit https://github.com/jimmyvanau/twofactorauth/commit/db02c583824c085695c38a46c9d22fbde52c3a6d

**Comment by Carlgo11** [comment]
@jimmyvanau could you try and [log in](https://circleci.com/login-public/) to CircleCI?
Maybe that fixes the issue :shrug:

**Comment by jimmyvanau** [comment]
Logged in via Github

**Comment by jimmyvanau** [comment]
Just checked
#6311 does not have a pull/6311 branch in CI, nor
#6313 #6292
🤔

**Comment by Carlgo11** [comment]
Now it works! :partying_face:

**Comment by jimmyvanau** [comment]
Given the other PRs referenced, is this something worth investigating? Maybe updating [`CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) to ask all new contributors to log into CircleCI? Happy to raise an issue to discuss.

**Comment by Carlgo11** [comment]
@jimmyvanau Making a note about it shouldn't be necessary. I don't know why the builds didn't work on your PRs before or why it suddenly worked after you signed in on CircleCI but as far as I know it hasn't happened to anyone else before you.

Anyway, the build on this PR is currently failing because the Facebook page you provided is restricted in some way. Since you've also provided a Twitter handle I suggest we stick with that one and remove the Facebook tag.

File: img/s/service.nsw.gov.au.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3-14.3 13.8-7.8 6.4-19.5 16.6-26 22.8-11.1 10.6-11.7 11.1-12.6 8.8-14.1-36.5-25.5-64.2-27.5-66.7-2.3-2.9 29.3-30.5 43.7-38.2 9.5-5.1 13-5.8 16.5-3.4zm113.7 42.9c4.3 4.3 6.2 15.5 7.1 43.1 1.3 37.9-3.4 94-11.1 133.6-6.4 32.7-18.1 67.7-29.2 87.6l-3.5 6.3h-13.9c-11.9 0-35.5-2.3-36.8-3.6-.2-.2 1-6.8 2.8-14.6 3.6-16.3 6.5-34.1 8.3-50.8 1.6-15 1.6-56.5 0-70.5-3.9-34-9.4-64.3-14.2-77.9-1.7-4.7-3-8.7-3-8.9 0-1.1 13.2-11 22.2-16.6 12.1-7.6 33.6-18.9 48-25.2 13.4-5.8 19.3-6.4 23.3-2.5zm-105.3 60.7c3.4 6.8 9.6 47.7 11.8 77.4 1.3 18.2 1.3 54.9-.1 69.5-1.9 21.4-7.5 52.8-9.8 55.3-1 1-25-7.3-36.5-12.7-13.1-6.1-26.5-14.2-38.5-23.1-10.7-8-32.8-28.7-35-32.9-1.2-2.2-.9-3.3 4.2-12.9 6.9-13.1 15.8-27.4 25.1-40.4 28.3-39.6 70.2-85.8 76.1-84 .6.2 1.8 1.9 2.7 3.8zM400.4 303c2.1.4 4.6 1.6 5.5 2.4 4.5 4.5-2.8 30.9-15.1 54.9-4.9 9.6-15.7 26.6-21.1 33.2l-2.4 2.9-13.9-6.5c-7.7-3.7-19.2-8.8-25.7-11.4-9.1-3.7-11.6-5.1-11.3-6.4 2.5-8.4 6.4-37.9 7.3-54.4.4-9.5 1-13 2-13.4 4.3-1.7 68.4-2.8 74.7-1.3zm-70.3 92.9c27.2 11.2 53.1 23.5 56.4 26.8 2.8 2.7 1.8 7.2-2.2 11-14.2 12.9-41.1 26.6-67.4 34.2-11 3.2-30.4 6.9-31.4 6-.3-.3 1.3-4.6 3.6-9.5 9.2-19.8 17.5-44.5 22.3-66.7 1.1-4.7 2.2-7.7 3-7.7.7 0 7.8 2.7 15.7 5.9z"/></svg>
```
Comment (phallobst) [inline img/s/service.nsw.gov.au.svg]: The transparent parts of the logo should be white instead.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><circle cx="256" cy="256" r="256" fill="#d7153a"/><path fill="#fff" d="M79.6 148.5c10.3 10.9 36.2 64.7 57.5 119.5 6.6 16.9 6.7 17.6 5 19.5-28.4 31.9-51.9 64.4-64.2 88.8l-3.1 6.1-2.5-2.9a247.6 247.6 0 0 1-21.1-44.6C36.7 291.4 31.4 241.7 38 212.5c6.2-27.6 20.7-57.9 31.2-65.3 4.4-3 6.5-2.8 10.4 1.3zm109.2 14c4.8 3.4 9.8 15.2 14.6 34a225 225 0 0 1 5.6 28.9c0 1.5-3.9 5.3-14.3 13.8a563.8 563.8 0 0 0-26 22.8c-11.1 10.6-11.7 11.1-12.6 8.8a841.5 841.5 0 0 0-27.5-66.7c-2.3-2.9 29.3-30.5 43.7-38.2 9.5-5.1 13-5.8 16.5-3.4zm113.7 42.9c4.3 4.3 6.2 15.5 7.1 43.1 1.3 37.9-3.4 94-11.1 133.6-6.4 32.7-18.1 67.7-29.2 87.6l-3.5 6.3h-13.9c-11.9 0-35.5-2.3-36.8-3.6-.2-.2 1-6.8 2.8-14.6a480 480 0 0 0 8.3-50.8c1.6-15 1.6-56.5 0-70.5-3.9-34-9.4-64.3-14.2-77.9-1.7-4.7-3-8.7-3-8.9 0-1.1 13.2-11 22.2-16.6a441.5 441.5 0 0 1 48-25.2c13.4-5.8 19.3-6.4 23.3-2.5zm-105.3 60.7c3.4 6.8 9.6 47.7 11.8 77.4 1.3 18.2 1.3 54.9-.1 69.5-1.9 21.4-7.5 52.8-9.8 55.3-1 1-25-7.3-36.5-12.7a227 227 0 0 1-38.5-23.1c-10.7-8-32.8-28.7-35-32.9-1.2-2.2-.9-3.3 4.2-12.9a360.8 360.8 0 0 1 25.1-40.4c28.3-39.6 70.2-85.8 76.1-84 .6.2 1.8 1.9 2.7 3.8zM400.4 303c2.1.4 4.6 1.6 5.5 2.4 4.5 4.5-2.8 30.9-15.1 54.9a253.9 253.9 0 0 1-21.1 33.2l-2.4 2.9-13.9-6.5a507 507 0 0 0-25.7-11.4c-9.1-3.7-11.6-5.1-11.3-6.4 2.5-8.4 6.4-37.9 7.3-54.4.4-9.5 1-13 2-13.4 4.3-1.7 68.4-2.8 74.7-1.3zm-70.3 92.9c27.2 11.2 53.1 23.5 56.4 26.8 2.8 2.7 1.8 7.2-2.2 11-14.2 12.9-41.1 26.6-67.4 34.2-11 3.2-30.4 6.9-31.4 6-.3-.3 1.3-4.6 3.6-9.5a335 335 0 0 0 22.3-66.7c1.1-4.7 2.2-7.7 3-7.7.7 0 7.8 2.7 15.7 5.9z"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**